cfme/services/myservice/ui.py

```diff
@@ -354,6 +354,7 @@ def download_file(self, extension):
 def reconfigure_service(self):
     # TODO refactor this method - it does nothing at the moment. Bug 1575935
     view = navigate_to(self, 'Reconfigure')
+    wait_for(lambda: view.submit_button.is_displayed, num_sec=5)
```

Comment:
Are you waiting for the submit button to be displayed or for it to be enabled? If the latter you must do 
```python
    wait_for(lambda: not view.submit_button.disabled, timeout=7)
```
If the former you can use `view.submit_button.wait_displayed()`

Reply:
done